tests/test_client.py

```diff
@@ -1679,6 +1679,16 @@ def test_override_flow_via_kwargs(self):
         self.assertEqual(client.OOB_CALLBACK_URN, q['redirect_uri'][0])
         self.assertEqual('online', q['access_type'][0])
 
+    def test__check_for_approval_prompt(self):
+        self.flow.params['approval_prompt'] = 'force'
+        self.flow._check_for_approval_prompt()
```

Comment:
Yes, sorry, I wasn't as done as I thought. :)

On Wed, Jul 27, 2016 at 2:48 PM Nathaniel Manista <notifications@github.com>
wrote:

> In tests/test_client.py
> <https://github.com/google/oauth2client/pull/572#discussion_r72530595>:
>
> > @@ -1679,6 +1679,16 @@ def test_override_flow_via_kwargs(self):
> >          self.assertEqual(client.OOB_CALLBACK_URN, q['redirect_uri'][0])
> >          self.assertEqual('online', q['access_type'][0])
> >
> > +    def test__check_for_approval_prompt(self):
> > +        self.flow.params['approval_prompt'] = 'force'
> > +        self.flow._check_for_approval_prompt()
>
> This method no longer exists, right?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/google/oauth2client/pull/572/files/ccddbc22ca51553980a099cf8ccaba839eeae71a#r72530595>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAPUc2YCv0KYIhId8fKvUMNbBanb77Nbks5qZ9IqgaJpZM4JWpSX>
> .
>